### PR TITLE
wrap send_key commands with better timeouts before assert_screen

### PR DESCRIPTION
**Pull request by srlemke**
- Related ticket: https://progress.opensuse.org/issues/57704
- Needles: None
- Verification run: https://openqa.suse.de/tests/overview?build=srlemke%2Fos-autoinst-distri-opensuse%23yast_apparmor&version=12-SP3&distri=sle

### CODE REVIEW

File: tests/console/yast2_apparmor.pm [hunk 1]
```diff
@@ -104,11 +104,13 @@ sub run {
...
+        wait_still_screen(stilltime => 5);
         if (is_pre_15()) {
-            wait_screen_change { send_key 'tab' };
-            wait_screen_change { send_key 'end' };    # we need to search for recent toggled element at the of the list
+            send_key 'tab';
+            wait_still_screen(stilltime => 5);
+            send_key 'end';
         }
-        wait_still_screen(stilltime => 2);
+        wait_still_screen(stilltime => 5);
```
Comment (okurz) [inline tests/console/yast2_apparmor.pm]: I wonder why there would be the need for a `wait_still_screen` at all. The subsequent `assert_screen` already waits for the screen to appear. Maybe the needle is not asserting that the screen is fully rendered yet?

File: tests/console/yast2_apparmor.pm [hunk 2]
```diff
@@ -104,11 +104,13 @@ sub run {
         wait_screen_change { send_key 'down' };
         save_screenshot;
         send_key(is_pre_15() ? 'alt-t' : 'alt-c');
+        wait_still_screen(stilltime => 5);
         if (is_pre_15()) {
-            wait_screen_change { send_key 'tab' };
-            wait_screen_change { send_key 'end' };    # we need to search for recent toggled element at the of the list
+            send_key 'tab';
+            wait_still_screen(stilltime => 5);
```
Comment (okurz) [inline tests/console/yast2_apparmor.pm]: An explicit `assert_screen` is always wasting less time than a `wait_still_screen`. This call takes *at least* 5 seconds now even when not necessary. This test inefficiency accumulates over many calls and also many modules.

Comment (srlemke) [reply tests/console/yast2_apparmor.pm]: Thanks for the tips, let me improve that!

File: tests/console/yast2_apparmor.pm (hunk 1 above)
Comment (srlemke) [reply tests/console/yast2_apparmor.pm]: Not sure, will test that. 
This test was 'unstable' (fails random only on SLE12.3 and only in 'production') so it had to be timeouts, but ill be happy to test some tuning downs.

File: tests/console/yast2_apparmor.pm (hunk 2 above)
Comment (srlemke) [reply tests/console/yast2_apparmor.pm]: Created a new PR. Thanks!

**Pull request merged**